package.json

```diff
@@ -140,6 +140,11 @@
     "precommit": "lint-staged",
     "formatAll":
       "find ./common/ -name '*.ts*' | xargs prettier --write --config ./.prettierrc --config-precedence file-override",
+    "prettier:diff-ts":
+      "prettier --write --config ./.prettierrc --list-different \"common/**/*.ts\"",
+    "prettier:diff-tsx":
+      "prettier --write --config ./.prettierrc --list-different \"common/**/*.tsx\"",
+    "prettier:diff": "npm run prettier:diff-ts && npm run prettier:diff-tsx",
```

Comment:
We don't need both `prettier:diff-tsx` and `prettier:diff-ts`, a single regex should be able to accommodate both.